src/constants/navbar-Content.ts

```diff
@@ -0,0 +1,53 @@
+export const navBarContentMock = [
+    {
+        title: 'Tasks',
+        refURL: '/',
+        pathName: '/',
+        pipeSymbol: '|',
+    },
+    {
+        title: 'Issues',
+        refURL: '/issues',
+        pathName: '/issues',
+        pipeSymbol: '|',
+    },
+    {
+        title: 'Mine',
+        refURL: '/mine',
+        pathName: '/mine',
+        pipeSymbol: '|',
+    },
+    {
+        title: 'Open PRs',
+        refURL: '/openPRs',
+        pathName: '/openPRs',
+        pipeSymbol: '|',
+    },
+    {
+        title: 'Stale PRs',
+        refURL: '/stale-pr',
+        pathName: '/stale-pr',
+        pipeSymbol: '|',
+    },
+    {
+        title: 'Idle Users',
+        refURL: '/idle-users',
+        pathName: '/idle-users',
+        pipeSymbol: '',
+    },
+];
+
+export const featureFlag = [
```

Comment:
please change the variable name to featureFlags the plural form

Reply:
fixed